### ISSUE TITLE
Ontology Queries structure

### ISSUE THREAD

**Issue by bwalsh** [ga4gh/g2p-team#7]
_Leaving this open for commentary/additions_
### Ontology Queries
- The 'ontologySource' is assumed to be equivalent to an Ontologies 'prefix'.  However, no agreement or mechanism exists to align the ontologySource string to a specific URI.  

_Recommend:_ collapsing ontologySource and identifier into a single field 'ontologyURI' 
#### Discussion

During the call @mellybelly mentioned there was an yaml file to provide a mapping of commonly used sources.

**Comment by bwalsh** [comment]
Hi Brian,
take a look at:
https://github.com/monarch-initiative/dipper/blob/master/dipper/curie_map.yaml

Also see:
https://github.com/cmungall/biocontext

cheers,
Melissa

**Comment by antbro** [comment]
Hi Brian
I can see the logic, but thinking of longer term and broader utility I would worry about situations in which only the prefix or suffix are needed (e.g., a network usage in which a single or a local ontology is agreed as the one to use). Baking these together compromises flexibility, keeping them separate does no real harm (does it?)

**Comment by bwalsh** [comment]
Agreed.  With Melissa's contribution above, it gives us an internal mapping of 'prefix' to 'ontology url'
